storage/libradb/src/ledger_store/mod.rs

```diff
@@ -98,16 +98,35 @@ impl LedgerStore {
     /// [`start_epoch`, `end_epoch`). If there is no more than `limit` results, this function
```

Comment:
Should we allow search in [start, end] instead of [start, end) since the end epoch may be the latest epoch and it could have an `epoch_change_ledger_info`.

Reply:
https://github.com/libra/libra/blob/38536ae729a6b23a23e41979fb4d9309cd24c995/storage/libradb/src/lib.rs#L588

the call site offsets this by calling `.next_block_epoch()`. 

That can be a separate fix, we need to think it through before touching the interface.